### DISCUSSION TITLE
Article: `nothing`

### DISCUSSION THREAD

**Discussion by Firgrep** [systemphil/sphil#63] [Hegel]
> I'm not convinced by this division that Houlgate makes between the indeterminacy of Being developing into Nothing, and the immediacy of Nothing developing into Being. But again, this is something I could do in my addition to the article.
Something else that has just struck me about Houlgate's account is that he really makes use of the mechanism of sublation, without referring to it. This idea that nothing becomes being because " for simple immediacy, at this stage in speculative logic, is all that being is understood to be" is clearly making use of the mechanism of sublation. 

@ridetoruin

**Comment by ghost** [comment]


**Reply by Firgrep** [reply]
Yes, it appears that way. How do you read the transition (in a way that does not fall prey to the third-figure and other fallacies)?
